lib/3scale/backend.rb

```diff
@@ -53,6 +53,7 @@
 require '3scale/backend/failed_jobs_scheduler'
 require '3scale/backend/transactor'
 require '3scale/backend/listener'
+require '3scale/backend/opentelemetry'
```

Comment:
This is only executed from the backend listener?

Reply:
I'd say it only makes sense for the listener, which is who listens to sinatra routes

Reply:
the worker is also importing this file. 

What about `lib/3scale/backend/server.rb` ?? I did not try myself, though. Looks like `lib/3scale/backend/listener.rb` is being imported by both worker and listener :shrug: 

Not important, though.. up to you.. not a request for change. The worker will not have the opt-in switch enabled, so it is harmless.

Reply:
Fixed: https://github.com/3scale/apisonator/pull/405/commits/a45900b1f8e9c19675a0996e8a0e6cfeee34a421